Decred Wallet/Utils/LocalizedStrings.swift

```diff
@@ -0,0 +1,401 @@
+    static let passwordStrength = NSLocalizedString("passwordStrength", comment: "")
+    static let confirmPassword = NSLocalizedString("confirmPassword", comment: "")
+    
+    /* Request Password */
+    static let enterPassword = NSLocalizedString("enterPassword", comment: "")
+    static let promptSpendingPassword = NSLocalizedString("promptSpendingPassword", comment: "")
+    static let promptStartupPassword = NSLocalizedString("promptStartupPassword", comment: "")
+    static let securityTypeNotSpecify = NSLocalizedString("securityTypeNotSpecify", comment: "")
+    static let password = NSLocalizedString("password", comment: "")
+    static let proceed = NSLocalizedString("proceed", comment: "")
+    static let passwordEmpty = NSLocalizedString("passwordEmpty", comment: "")
+    static let invalidInput = NSLocalizedString("invalidInput", comment: "")
+    static let passwordInvalid = NSLocalizedString("passwordInvalid", comment: "")
+    
+    /* Request PIN and PIN Setup */
+    static let createPIN = NSLocalizedString("createPIN", comment: "")
+    static let enterPIN = NSLocalizedString("enterPIN", comment: "")
+    static let confirmPIN = NSLocalizedString("confirmPIN", comment: "")
+    static let pinsDidNotMatch = NSLocalizedString("pinsDidNotMatch", comment: "")
+    static let pin = NSLocalizedString("pin", comment: "")
+    static let pinStrength = NSLocalizedString("pinStrength", comment: "")
+    
+    /* Change Spendind PIN/Password */
+    static let currentSpendingPassword = NSLocalizedString("currentSpendingPassword", comment: "")
+    static let changingSpendingPINPass = NSLocalizedString("changingSpendingPINPass", comment: "")
+    
+    /* Changing Startup PIN/Password */
+    static let changingStartupPINPass = NSLocalizedString("changingStartupPINPass", comment: "")
+    static let removingStartupPINPass = NSLocalizedString("removingStartupPINPass", comment: "")
+    static let settingStartupPINPass = NSLocalizedString("settingStartupPINPass", comment: "")
+    
+    /* Transaction List */
+    static let pending = NSLocalizedString("pending", comment: "")
+    static let confirmed = NSLocalizedString("confirmed", comment: "")
+    static let vote = NSLocalizedString("vote", comment: "")
+    static let ticket = NSLocalizedString("ticket", comment: "")
+    static let confirmedLive = NSLocalizedString("confirmedLive", comment: "")
+    static let confirmedImmature = NSLocalizedString("confirmedImmature", comment: "")
+    static let noTransactions = NSLocalizedString("noTransactions", comment: "")
+    static let sent = NSLocalizedString("sent", comment: "")
+    static let received = NSLocalizedString("received", comment: "")
+    static let yourself = NSLocalizedString("yourself", comment: "")
+    static let staking = NSLocalizedString("staking", comment: "")
+    static let all = NSLocalizedString("all", comment: "")
+    
+    /* CreateNewWalletViewController */
```

Comment:
Please re-arrange these strings in the other that the pages appear when the app is launched. Makes it easier to scan through and determine where a certain string could be found.

Reply:
This will be in the next PR